bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/SetupListener.java

```diff
@@ -56,7 +56,7 @@ private boolean compatible(Alloc pred, Alloc succ) {
 		return 
 			pred != null &&
 			pred.getObs().getInstrumentString().equals(succ.getObs().getInstrumentString()) &&
-			pred.getObs().getCoords(pred.getMiddlePoint()).equals(succ.getMiddlePoint());
```

Comment:
I think we need either `pred.getEnd` or `succ.getStart` (i.e., the same time) for both observations.  We're trying to see whether setup is needed when switching from one observation to another so we want to know if they are in different places at the time of the switchover from one to another.

I'm guessing here, it would be good to get verification from @andrewwstephens.  It may be that we want `pred.getEnd` for the first observation and `succ.getStart` for the second and if they are not the same time and it is a fast moving non-sidereal then we need setup anyway even if the target is the same.